docs-requirements.txt

```diff
@@ -1,5 +1,5 @@
 docutils>=0.15.2,<0.17
-pydata-sphinx-theme>=0.3.1
+pydata-sphinx-theme>=0.7.2
```

Comment:
Required to fix this issue: 
https://github.com/numpy/numpy/issues/20690

Otherwise doc-builds continued to fail